docs/options-in-versions

```diff
@@ -257,6 +257,7 @@
 --trace-config                       8.3.0
 --trace-ids                          8.2.0
 --trace-time                         7.14.0
+--type-of-service                    8.8.0
```

Comment:
It is too late for 8.8.0 but it should be fit for 8.9.0.
```suggestion
--type-of-service                    8.9.0
```

Reply:
Done